src/resources/cdm.properties

```diff
@@ -172,8 +173,8 @@ spark.cdm.autocorrect.mismatch                    false
 #-----------------------------------------------------------------------------------------------------------
 spark.cdm.perfops.numParts                        10000
 spark.cdm.perfops.batchSize                       5
-spark.cdm.perfops.readRateLimit                   20000
-spark.cdm.perfops.writeRateLimit                  40000
+spark.cdm.perfops.ratelimit.origin                20000
```

Comment:
These would be better as `spark.cdm.perfops.origin.ratelimit` and `spark.cdm.perfops.target.ratelimit` to follow other naming conventions used here for properties.

Reply:
We both read and write from target, so I'd suggest this is cleaner. I don't think we particularly care about the distinction between "read" and "write", just a general "operations limit" applied to a cluster.

Reply:
at issue is the `.connection` property name was not consistent with the less-to-more-specific paradigm that all the other properties (including this one) addressed. I've pushed a change to this PR to address that.